tests/scripts/travis_scripts.sh

```diff
@@ -1,15 +1,40 @@
 #!/bin/bash
 
 # Common checks to get run during the 'script' section in Travis.
+OUTPUT=0
+
+# Make OUTPUT equal return code if return code is not 0
+function checkReturn {
```

Comment:
This is a great find. I can't believe we weren't testing this all along... 😔

I wanted to mention that perhaps we should be using [`set -e`](https://stackoverflow.com/questions/821396/aborting-a-shell-script-if-any-command-returns-a-non-zero-value) instead, which would just fail as soon as something failed. 

Either way works, I just wanted to mention this. 🤷‍♂️

Reply:
I kinda like it continuing to run, just to find all the problems so one can fix them all (hopefully) in a single commit. But I am open to changing it.

Reply:
That is a very good point, makes sense to me. I'll make as resolved, I just thought I'd mention it. I don't feel strongly one way or another.